mainhandler/vulnscan.go

```diff
@@ -532,9 +536,9 @@ func (actionHandler *ActionHandler) getImageScanCommand(containerData *utils.Con
 		cmd.Args[identifiers.AttributeUseHTTP] = true
 	}
 
-	// Add instanceID only if not empty
-	if containerData.InstanceID != "" {
-		cmd.InstanceID = &containerData.InstanceID
+	// Add instanceID only if container is not empty
+	if containerData.Slug != "" {
```

Comment:
Restores the behaviour that was changed in https://github.com/kubescape/operator/commit/bf8a74795b7e163eaf76c655a86f91d40ce62bc4

Reply:
It used to be like this: https://github.com/kubescape/operator/blob/v0.2.49/mainhandler/vulnscan.go#L517